src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -61,6 +62,7 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
             Engine = engine;
             _libraryLoaded = new List<string>();
             _loadOrder = 0;
+            _waitLoop = waitLoop;
```

Comment:
>_waitLoop [](start = 12, length = 9)

I wonder if we should avoid storing this in a member variable when it shouldn't be used after launch. If we do want it in a member variable, I think we want to make sure we clear it once Initialize returns.